taskcluster/ci/beetmover/kind.yml

```diff
@@ -37,7 +37,7 @@ tasks:
         worker:
             chain-of-trust: true
             max-run-time: 1800
-        run-on-tasks-for: [github-release, action:release-promotion]
```

Comment:
Technically we could do leave the promote phase to be triggered on release creation, and then we'd only have a `ship-client` phase in the action. But that's a bit clunky.. Just calling it out in case the VPN team would prefer that though.